src/community/gsr/src/main/java/org/geoserver/gsr/api/GeoServicesExceptionResolver.java

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2017 Boundless - http://boundlessgeo.com All rights reserved.
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
```

Comment:
Please update headers as appropriate, if you are donating this as a code grant you can preserve the above header after the osgeo header.

Reply:
fixed